executor/join.go

```diff
@@ -244,6 +246,11 @@ func (e *HashJoinExec) fetchBuildSideRows(ctx context.Context, chkCh chan<- *chu
 			e.buildFinished <- errors.Trace(err)
 			return
 		}
+		failpoint.Inject("errorFetchBuildSideRowsMockOOMPanic", func(val failpoint.Value) {
```

Comment:
`failpoint.Inject("errorFetchBuildSideRowsMockOOMPanic", nil)` and `c.Assert(failpoint.Enable("github.com/pingcap/tidb/executor/errorFetchBuildSideRowsMockOOMPanic", `panic`), IsNil)`